pkg/fftm/transaction_management.go

```diff
@@ -85,6 +85,20 @@ func (m *manager) getTransactions(ctx context.Context, afterStr, limitStr, signe
 
 }
 
+func (m *manager) updateTransaction(ctx context.Context, txID string, txUpdate *apitypes.TXUpdatesExternal) (transaction *apitypes.ManagedTX, err error) {
+	if txUpdate == nil {
+		return nil, i18n.NewError(ctx, tmmsgs.MsgInvalidSortDirection)
+	}
+	updatedTx, err := m.txHandler.HandleTransactionUpdate(ctx, txID, *txUpdate)
```

Comment:
Ok - so the whole of the job of thread management etc. is delegated down into the policy engine implementation 👍